website/content/docs/installing/no-gen-resources.mdx

```diff
@@ -0,0 +1,478 @@
+kms "aead" {
+	purpose   = "recovery"
+	aead_type = "aes-gcm"
+	key       = "8fZBjCUfN0TzjEGLQldGY4+iE9AkOvCfjh7+p0GtRBQ="
+	key_id    = "global_recovery"
+}
+EOT
+}
+```
+
+</Tab>
+</Tabs>
+
+
+## Initialize the Database
+
+Before you can start Boundary, the database must be initialized. To do this, run:
+
+```
+$ boundary database init -h
+...
+```
+
+From this command, you can see the flags available to skip the creation of auto-generated resources. These
+resources are useful when running in development environments but in a non development environment may not
+be desired. You can think of them as Boundary's default username and password, and if used in non dev workflows
+should be overidden or skipped to avoid undermining Boundary's security model.
+
+To initialize the Boundary database without generated resources:
+
+```
+$ boundary database init -skip-initial-login-role -config /etc/boundary.hcl
+```
+
+When you start Boundary, you will effectively have a blank sheet to work against. The initial migrations in the database have been run (note that this includes creating special users like `u_anon` and the `global` scope) and the internal keyrings have been initialized.. From here, it's required that
+you use the KMS recovery workflow described above to create at a minimum an auth method, a user, an account, and a 
+role with sufficient grants. Otherwise, you need to continue to use the recovery workflow for management. t's important 
+to realize that this is effectively a global super user type of workflow and comes with security concerns.
+
+## Creating Your First Login Account
+
+This section covers how to configure your first auth method, user, account, and role to login to Boundary without
+the recovery KMS workflow. In this example, we're going to make an admin user for the global and project level
+scopes we create. This will allow our user to configure targets within those scopes and manage them.
+
+#### Create Org and Project Scopes
+
+In this example, we're going to create an org and project scope and skip creating an administrator and admin role 
+for each scope. We're going to specify a role for managing these scopes by selected users in a later step.
+
+<Tabs>
+<Tab heading="CLI">
+
+```bash
+$ export BOUNDARY_RECOVERY_CONFIG=<<EOT
+kms "aead" {
+	purpose   = "recovery"
+	aead_type = "aes-gcm"
+	key       = "8fZBjCUfN0TzjEGLQldGY4+iE9AkOvCfjh7+p0GtRBQ="
+	key_id    = "global_recovery"
+}
+EOT
+
+$ boundary scopes create -name 'org' -scope-id 'global' \
+  -skip-admin-role-creation \
+  -skip-default-role-creation
+<truncated>
+
+$ boundary scopes create -name 'project' -scope-id <org_scope_id_from_last_step> \
+  -skip-admin-role-creation \
+  -skip-default-role-creation
+<truncated>
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+```hcl
+resource "boundary_scope" "org" {
+  scope_id    = "global"
+  name        = "organization"
+  description = "Organization scope"
+
+  auto_create_admin_role   = false 
```

Comment:
These are the defaults, you don't need to specify them. Not doing so makes it more clear in the other examples when you _are_ specifying them that that is not the default behavior.

Reply:
I went back and forth on this, I wanted to explicitly call them out since the logic on the CLI is other way around: in TF, it's the default but on the CLI, you have to explicitly call the flag. If someone was looking at the TF and CLI examples side by side, it would be more clear.

Reply:
Fair enough